nbbuild/antsrc/org/netbeans/nbbuild/CustomJavac.java

```diff
@@ -395,16 +410,20 @@ public boolean execute() throws BuildException {
                         }
                     }
                 }
-                if ("-Werror".equals(args[i])) {
-                    args[i] = "-Xlint:none";
-                }
             }
+            // nbjavac in version 19 contains invalid ct.sym files, which cause
+            // warnings from build. Some of the modules are compiled with
+            // -Werror and thus this breaks the build
+            // Ater the update to version 20 this should be removed.
+            String[] args2 = new String[args.length + 1];
+            args2[0] = "-Xlint:-classfile";
+            System.arraycopy(args, 0, args2, 1, args.length);
```

Comment:
Thank you @matthiasblaesing - this is certainly better than my previous `-Xlint:none` code!